serverManager/service/source/ConfigHelper.cpp

```diff
@@ -126,14 +129,18 @@ void ConfigHelper::readConfigFile(const std::string &filePath)
         return;
     }
 
-    std::map<std::string, std::string> envVariables{convertToMap(configReader->getEnvironmentVariables())};
-    for (const auto &[name, value] : envVariables)
+    // Always override env variables when present in "more important" file
+    // (envVariables + extraEnvVariables from less important file will be wiped if envVariables are present)
```

Comment:
I like this comment.
Therefore, would "lowerPriorityEnvVariables" be a better name than "extraEnvVariables" ?

Reply:
again, comcast decided to have extraEnvVariables, we can't do anything with it